manifests/l/Luca-AlexanderThomas/AutoHome-Autostart/1.0.0/Luca-AlexanderThomas.AutoHome-Autostart.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Luca-AlexanderThomas.AutoHome-Autostart
+PackageVersion: 1.0.0
+Installers:
+- InstallerLocale: de-DE
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/luca-alexander-thomas/AutoHome/releases/download/1.0/AutoHome-Installer-1.0.0_silent-install.msi
+  InstallerSha256: 7A2318E0856A5CCF2CCAAC7852EA4521617C6EC71FF4F5031A6A1AEDF0F417BE
+  ProductCode: '{DCF0C8FF-8909-4BBD-9147-5A5003E7464E}'

```

```suggestion
  ProductCode: '{DCF0C8FF-8909-4BBD-9147-5A5003E7464E}'
```
winget uninstall "{DCF0C8FF-8909-4BBD-9147-5A5003E7464E}" does not work.

---

ok are there no more mistakes in the Configuration?